PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Inexor <https://discord.gg/Sz4SKXpxRN>
+
+buildarch=
+
+pkgname="inexor-git"
+pkgver=0.1.3
```

Comment:
Which would refer to the engine version, but you build the application which has a different version.